google/gke/gke-how-to/deploy-pro-gke-cluster.rst

```diff
@@ -0,0 +1,137 @@
+Deploy a Pro GKE Cluster
+========================
+
+
+Limitations
+-----------
+
+Google does not yet have Ubuntu Pro image offerings for the GKE nodes i.e. you cannot choose
+Ubuntu Pro images for the GKE nodes. GKE does not support custom images for the nodes either.
+Additionally, GKE does not allow for post-deployment customisation of node VMs.
+
+	"Modifications on the boot disk of a node VM do not persist across node re-creations.
+	Nodes are re-created during manual upgrade, auto-upgrade, auto-repair, and auto-scaling.
+	In addition, nodes are re-created when you enable a feature that requires node re-creation,
+	such as GKE Sandbox, intranode visibility, and shielded nodes."
+
+	-- `GKE docs <https://cloud.google.com/kubernetes-engine/docs/concepts/node-images#modifications>`_
+
+Given that, currently, there's no mechanism to either enable Ubuntu Pro or pre-bake the UA token
+in the specific cluster, so the managed Kubernetes is not applicable for the GCE.
+
+
+Deploy Kubernetes manually on Ubuntu Pro VMs
+--------------------------------------------
+
+One solution to have a Ubuntu Pro cluster is to deploy and manage Kubernetes manually on
+Ubuntu Pro VMs available in GCP.
```

Comment:
```suggestion
Ubuntu Pro VMs available in GCE.
```

GCP was correct, but I think GCE makes as much sense here and it avoids using too many G** acronyms